frigate/detectors/detection_api.py

```diff
@@ -9,8 +13,67 @@ class DetectionApi(ABC):
 
     @abstractmethod
     def __init__(self, detector_config):
-        pass
+        self.detector_config = detector_config
+        self.thresh = 0.5
+        self.height = detector_config.model.height
+        self.width = detector_config.model.width
 
     @abstractmethod
     def detect_raw(self, tensor_input):
         pass
+
+    def yolonas(self, output):
```

Comment:
maybe this should be called `post_process_yolonas` so the meaning is more clear?